frontend/src/components/wallet/StakeAllocationBlock.vue

```diff
@@ -82,7 +82,9 @@ export default {
       //   0
       // )
 
-      const colors = ['#94DE69', '#4CB7F9', '#6864FF', '#8E2ADF', '#CC3ED0', '#FD5EAB', '#FFA267', '#F0E466']
+      const colors = ['#94DE69', '#4CB7F9', '#6864FF', '#8E2ADF', '#CC3ED0', '#FD5EAB', '#FFA267', '#F0E466',
```

Comment:
Is 24 delegation maximum?

Reply:
May be we should add color generator function?

Reply:
Good point... I believe the colors actually cycle themselves but just in case I doubled the amount of colors for now. Edge case I believe